src/graphql/resolvers/Mutation/approveCharterFields.js

```diff
@@ -89,6 +91,19 @@ export default async (
 
 		if (canBeActive) {
 			org.active = true;
+
```

Comment:
It seems that this code is duplicated a bit, do you think it would be worthwhile to put it into a function?

Reply:
It could be worthwhile...
maybe a function that accepts the settings object and an organization object and checks to see whether or not its locked? don't know. don't see any other settings that we would need in the near future.

Reply:
Sounds like a good form, would you like me to implement it?